### PR TITLE
[core-v2.8.4-alpha.2, react-v2.3.4-alpha.2, vue-v2.2.4-alpha.2] : Enhancement - Improved theme nomenclature

### PR DESCRIPTION
**Pull request by Adamj1232**
### Description
This PR focuses on created variables that will allow for theming along with more customized styling by developers.
In the demo app there is the theming abstraction to be tested with by @gesquinca 
Added variables to allow for more in-depth and customized theming. 
Added a component to the internal demo to help style that uses a mapping to a smaller subset of variables.

Added variables: 
```
  --account-center-border
  --account-center-box-shadow
  --account-center-boarder-radius
  --account-center-chain-warning
  --account-center-minimized-balance-color
  --account-center-minimized-chain-select-background
  --account-center-maximized-network-section-background
  --account-center-maximized-network-text-color
  --account-center-maximized-info-section-background-color
  --account-center-maximized-upper-action-color
  --account-center-maximized-upper-action-background-hover
  --account-center-maximized-app-name-color
  --account-center-maximized-app-info-color
  --account-center-micro-background
  
    --notify-onboard-z-index
  --notify-onboard-background
  --notify-onboard-close-icon-color
  --notify-onboard-close-icon-hover
  --notify-onboard-transaction-status-color
  --notify-onboard-transaction-font-size
  --notify-onboard-hash-time-font-size
  --notify-onboard-hash-time-font-line-height
  --notify-onboard-address-hash-color
  --notify-onboard-anchor-color
  ```

### Checklist
- [x] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [x] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works
- [x] I have run `yarn type-check` & `yarn build` to confirm there are not any associated errors
- [x] This PR passes the Circle CI checks

### CODE REVIEW

File: packages/core/README.md [hunk 1]
```diff
@@ -839,6 +841,23 @@ The Onboard styles can customized via [CSS variables](https://developer.mozilla.
   --account-center-app-btn-background
   --account-center-app-btn-font-family
 
+  --account-center-border
+  --account-center-box-shadow
+  --account-center-boarder-radius
```
Comment (aaronbarnardsound) [inline packages/core/README.md:846]:
```suggestion
  --account-center-border-radius
```

File: packages/core/src/views/account-center/Maximized.svelte [hunk 2]
```diff
@@ -54,7 +54,10 @@
       --account-center-maximized-upper-background,
       var(--onboard-gray-600, var(--gray-600))
     );
-    border-radius: var(--onboard-border-radius-3, var(--border-radius-3));
+    border-radius: var(
+      --account-center-boarder-radius,
```
Comment (aaronbarnardsound) [inline packages/core/src/views/account-center/Maximized.svelte:58]:
```suggestion
      --account-center-border-radius,
```

File: packages/core/src/views/account-center/Maximized.svelte [hunk 3]
```diff
@@ -124,10 +136,16 @@
   }
 
   .network-container {
-    border-radius: var(--onboard-border-radius-3, var(--border-radius-3));
+    border-radius: var(
+      --account-center-boarder-radius,
```
Comment (aaronbarnardsound) [inline packages/core/src/views/account-center/Maximized.svelte:140]:
```suggestion
      --account-center-border-radius,
```

File: packages/core/src/views/account-center/Maximized.svelte [hunk 4]
```diff
@@ -63,7 +66,10 @@
 
   .wallets-section {
     width: 100%;
-    border-radius: var(--onboard-border-radius-3, var(--border-radius-3));
+    border-radius: var(
+      --account-center-boarder-radius,
```
Comment (aaronbarnardsound) [inline packages/core/src/views/account-center/Maximized.svelte:70]:
```suggestion
      --account-center-border-radius,
```

File: packages/core/src/views/account-center/Maximized.svelte [hunk 5]
```diff
@@ -147,36 +165,54 @@
...
-      --account-center-maximized-app-info-section,
-      var(--onboard-white, var(--white))
+      --account-center-maximized-info-section-background-color,
+      var(
+        --account-center-maximized-info-section,
+        var(--onboard-white, var(--white))
+      )
+    );
+    border-radius: var(
+      --account-center-boarder-radius,
```
Comment (aaronbarnardsound) [inline packages/core/src/views/account-center/Maximized.svelte:175]:
```suggestion
      --account-center-border-radius,
```

File: packages/core/src/views/account-center/Micro.svelte [hunk 6]
```diff
@@ -17,15 +17,25 @@
...
+      var(--onboard-shadow-3, var(--shadow-3))
+    );
     width: 80px;
     pointer-events: auto;
   }
 
   .radius {
-    border-radius: var(--onboard-border-radius-3, var(--border-radius-3));
+    border-radius: var(
+      --account-center-boarder-radius,
```
Comment (aaronbarnardsound) [inline packages/core/src/views/account-center/Micro.svelte:36]:
```suggestion
      --account-center-border-radius,
```

File: packages/core/src/views/account-center/Minimized.svelte [hunk 7]
```diff
@@ -62,14 +62,21 @@
...
+      --account-center-box-shadow,
+      var(--onboard-shadow-3, var(--shadow-3))
+    );
     pointer-events: auto;
   }
 
   .radius {
-    border-radius: var(--onboard-border-radius-3, var(--border-radius-3));
+    border-radius: var(
+      --account-center-boarder-radius,
```
Comment (aaronbarnardsound) [inline packages/core/src/views/account-center/Minimized.svelte:77]:
```suggestion
      --account-center-border-radius,
```

**Review by aaronbarnardsound — APPROVED** [review]
Nice! Just a spelling fix suggestion, but otherwise good to go.

**Comment by Adamj1232** [comment]
> Nice! Just a spelling fix suggestion, but otherwise good to go.

@aaronbarnardsound I swear I know how to spell boarder 🏂 lol 🤦‍♂️

**Pull request merged**